wiki/en/Server-Troubleshooting.md

```diff
@@ -23,29 +23,29 @@ permalink: "/wiki/Server-Troubleshooting"
+If you are registered OK (you can [see it here](https://explorer.jamulus.io/)) and you or your friends can't see your Server, you may need to wait, or start your Client with the `--showallservers` option and try connecting from there ([see command line options](Software-Manual#command-line-options)) on how to start your Client with a config option). In some network configurations, you may also need to [port forward](Running-a-Server#port-forwarding) your router.
 
 If you are seeing a message that says the Server is full, you will need to wait until a slot becomes free.
 
 You can verify that your Server is listed in the relevant genre by [checking it here](https://explorer.jamulus.io/).
 
 ## Servers - Unregistered 
 
-### I'm running my client on the same machine/network as my Server but I can't connect to it
+### I'm running my Client on the same machine/network as my Server but I can't connect to it
 
-Connect your client to `127.0.0.1` (or `localhost` if that works for you). If you're running the client on a different machine to the Server but on the same network, then connect to the _local network_ address of the Server. Do not connect via the Server's public (WAN) address.
+Connect your Client to `127.0.0.1` (or `localhost` if that works for you). If you're running the Client on a different machine to the Server but on the same network, then connect to the _local network_ address of the Server. Do not connect via the Server's public (WAN) address.
 
 ### What address do I give to people so that they can connect to my Server?
 
-This should be your **public** IP address (find that with [Google](https://www.google.com/search?q=whatsmyip)). Connect your **own** Jamulus client to the **local** IP address of your Server (127.0.0.1 if it's on the same machine as your client). Note that your public IP address can change - see the note on "dynamic DNS" in [this guide](Running-a-Server#dynamic-dns-and-why-you-will-probably-need-it).
+This should be your **public** IP address (find that with [Google](https://www.google.com/search?q=whatsmyip)). Connect your **own** Jamulus Client to the **local** IP address of your Server (127.0.0.1 if it's on the same machine as your Client). Note that your public IP address can change - see the note on "dynamic DNS" in [this guide](Running-a-Server#dynamic-dns-and-why-you-will-probably-need-it).
 
 ### What port numbers can I use?
 
 You can set your Server to listen on a custom port with the `--port` option. If you do this, you will need to tell people which port to connect on. They will need to append the port number to the address of your Server in the format `[serverAddress]:[portNumber]`.
```

Comment:
Hm.  Again, not your change...  I'd like **Server Address** to mean "**hostname or IP number**, optionally followed by ":" and a **port number**".  That way we can explain the format _once_, explain when the ":port" can be omitted _once_ and explain when an **IP number** might be needed _once_.  Then just reference that explanation wherever **Server Address** is mentioned (i.e. linkified).

Reply:
We can't linkify it without making it a heading style, which would put it all on a different line. We _might_ be able to hack and anchor with a CSS style, but I don't know how to do that. Unless it's going to change, we may as well repeat it every time if we think people are confused about IPs vs names and :ports.

Reply:
Maybe adding `{#id}` or `{:#id}` works. I‘d look in the liquid or Jekyll docs.